app/services/faraday_helpers.rb

```diff
@@ -23,8 +23,15 @@ def stream_xml(url, verb = :get, options = {})
   # TODO: Add custom user_agent, timeout, proxies, and retries
   # TODO: Add block handling for stream_xml
 
+  def connection
```

Comment:
Should this sit here or in a faraday config file? @patanj101 can you also provide a view

Reply:
Good question. We don't currently have any config file for Faraday afaik.